components/AuthorAvatars.js

```diff
@@ -1,5 +1,8 @@
-import {buildImageUrl } from 'cloudinary-build-url'
+import {setConfig,buildImageUrl } from 'cloudinary-build-url'
 
+setConfig({
```

Comment:
@derberg we need to add the cloudinary cloudname here

Reply:
@derberg I researched. I did not find use of the API key and secret to configure cloudinary's image transformation service. 
I think that the service is a part of the free plan and does not require API secret. In the free tier the size of a single image should be within 10 MB. [referred this](https://support.cloudinary.com/hc/en-us/articles/202520592-Do-you-have-a-file-size-limit-#:~:text=On%20our%20free%20plan%2C%20the,file%20size%20is%20100%20MB.)
Should we use the service? I don't think there would be any security issues as the service is free for all.

Reply:
cool, 10MB is more than enough

so after adding the account name do you see it working in the preview?

Reply:
@derberg
Yup I saw the preview and used lighthouse for analytics
The solution seems to be improving the analytics of the blog page for the web but not for mobile. Will look into it and get back to you within a day with a better solution.
![image](https://user-images.githubusercontent.com/86974918/223858932-742d515a-41d6-469f-91a6-194c79c4d272.png)